doc/content/examples/example11.md

```diff
@@ -0,0 +1,104 @@
+| - | - | - | - |
+| 1 | 4.1956e-05 | 18.58 | 0.1 |
+| 2 | 4.4655e-05 | 13.34 | 0.11 |
+| 4 | 4.5735e-05 | 11.25 | 0.141 |
+| 8 | 4.6060e-05 | 10.62 | 0.188 |
+
+
+!table id=static_results_quadratic caption= HEX20 (Quadratic) elements refinement study.
+| Elements | Displacement (m) | Error (%) | Run time (s) |
+| - | - | - | - |
+| 1 | 5.1020e-05 | 0.99 | 0.207 |
+| 2 | 5.1696e-05 | 0.32 | 0.339 |
+
+It is worth noting that HEX27 elements were also tested and they produced nearly identical results to those of HEX20 elements.
+
+## Dynamic Loading
+
+Similar to the example shown above, this model utilizes the same beam, and follows the same mesh refinement strategy, the difference is that this beam is subject to an Ormsby Wavelet at one end and free at the other end. The response is measured at the free end in order to determine the natural frequency of the beam. The first natural frequency of the beam can be computed using the analytical expression shown below:
+
+\begin{equation}
+f_{n_f} = \frac {1.875^2}{2\pi}\sqrt{\frac {EI}{\rho AL^4}} = 18.65Hz
+\end{equation}
+
+The Ormsby wavelet is created using the [OrmsbyWavelet](functions/OrmsbyWavelet.md) function type.
+
+!listing examples/ex11/dynamic/3d_beam_dynamic.i
+         start=Functions
+         end=Materials
+
+The wavelet is applied as an acceleration at the fixed end of the beam using the [PresetAcceleration](bcs/PresetAcceleration.md) boundary condition.
+
+!listing examples/ex11/dynamic/3d_beam_dynamic.i
+         start=BCs
+         end=Functions
+
+The simulation results are captured using [VectorPostprocessors](VectorPostprocessors/index.md), specifically the [ResponseHistoryBuilder](vectorpostprocessors/ResponseHistoryBuilder.md) and [ResponseSpectraCalculator](vectorpostprocessors/ResponseSpectraCalculator.md)
+
+!listing examples/ex11/dynamic/3d_beam_dynamic.i
+         start=VectorPostprocessors
+         end=Outputs
+
+!media media/examples/ex11/dynamic_beam_plot.png
+         style=width:80%;margin-left:120px;float:left;
+         id=fig:dynamic_beam
+         caption=Acceleration response spectrum plot for dynamic beam simulation
+
+Simulation time for each of the models was recorded as well as the resulting first natural frequencies.
+
+!table id=static_results_linear caption= HEX8 (Linear) elements refinement study.
+| Elements | Mode 1 (Hz) | Run time (s)|
+| - | - | - |
+| 1 | 20.24 | 3.206 |
+| 2 | 19.73 | 5.376 |
+| 4 | 19.40 | 9.788 |
+| 8 | 19.39 | 18.282 |
+
+
+!table id=static_results_quadratic caption= HEX20 (Quadratic) elements refinement study.
+| Elements | Mode 1 (Hz) | Run time (s) |
+| - | - | - |
+| 1 | 18.41 | 23.333 |
+| 2 | 18.42 | 46.538 |
+
+This example was created to provide some insight into how bending is captured using continuum elements. For both of these static and dynamic simulations, the second order elements provide a result that is more consistent with the analytical solutions, although the computation cost is much higher.
```

Comment:
Can you also mention somewhere that the frequencies are calculated from the peaks of the Ormsby wavelet?